core/components/atoms/metricInput/__stories__/withSuffix.story.jsx

```diff
@@ -24,13 +25,20 @@ export const withSuffix = () => {
   );
 };
 
-const customCode = `() => {
+const customCode = `/*
+// style.css
+.MetricInput-wrapper--small {
+    width: var(--spacing-6);
+}
```

Comment:
Please use `Col--1` className or find some other better way to control the width